tests/functional/spec/list_collector_section_summary_items.spec.js

```diff
@@ -0,0 +1,139 @@
+import AnyCompaniesOrBranchesDrivingQuestionPage from "../generated_pages/list_collector_section_summary_items/any-companies-or-branches.page.js";
```

Comment:
- Some duplication in this that could be reduced. It also a bit crammed therefore not very readable and hard to follow. Consider breaking into multiple describes.
- Need to test the anchor link.
- We need a test where the collector isn't on the path. (Can be unit test)
   - One should be where you add items, then make it not on the path. So a list with answers won't show unless on the path.
- We need one for list collector is on path but empty (test placeholder text)
- Need to show driving question is shown on the summary.

Reply:
I think all the scenarios are covered in func test now, reduced it by moving repeating actions into its own functions.

Reply:
There is a bug with `Using the add link to add more items from the summary which should loop you back into the list collector until you say No.`. We'll look at it separately.